docs/provisioning/silo_open.md

```diff
@@ -0,0 +1,48 @@
+# Create an open sandbox silo
+
+:warning: This should be used for **development purpose only**.
+
+## Prerequisites
+
+To run these deployment options, you first need:
+- an existing Azure ML workspace (see [cookbook](README.md))
```

Comment:
Coud this link point directly to the workspace section? /docs/provisioning/README.md#create-an-azure-ml-workspace